pymarkdown/core.py

```diff
@@ -50,7 +56,11 @@ def step(part, scope, state):
     if isinstance(part, (str, unicode)):
         return [part], scope, state
     if isinstance(part, doctest.Example):
-        if valid_statement('_last = ' + part.source):
+        if isprint(part.source):
+            with swap_stdout() as s:
+                exec(part.source, scope)
+            result = s.getvalue().rstrip().strip("'")
```

Comment:
Hrm, this feels a bit overly specialized.  I wonder if, instead, we should always emit `stdout`, regardless of whether it is a print statement or not.